### PR TITLE
Add distinct on clause

### PR DESCRIPTION
**Pull request by mkangia**
Need distinct on for some work on UCRs.

### CODE REVIEW

File: tests/test_columns.py [hunk 1]
```diff
@@ -222,3 +223,21 @@ def _get_view_data(self, view):
...
+            distinct=['user', 'year'],
+            order_by=[OrderBy('user'), OrderBy('year'), OrderBy('date', is_ascending=False)],
+            group_by=['user', 'date']
+        )
+        vc.append_column(SimpleColumn('user'))
+        vc.append_column(YearColumn('date', alias='year'))
+        vc.append_column(SimpleColumn('indicator_a'))
+        result = vc.resolve(self.session.connection())
+        self.assertEquals(result, {
+            ('user1', datetime.date(2013, 2, 1)): {'user': 'user1', 'year': 2013.0, 'indicator_a': 3,
```
Comment (orangejenny) [inline tests/test_columns.py]: It's weird that a year value has a `.0`.

Comment (mkangia) [reply tests/test_columns.py]: I think some parsing is done later but ya its weird.

**Comment by mkangia** [comment]
thanks for the review @orangejenny 
I usually bump version post approval to be sure it would go through, updated now.

File: CHANGES.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 History
 =========
 
+0.16.0 (2020-03-24)
+----------------
+* Added support for distinct on (#66)
```
Comment (orangejenny) [inline CHANGES.md]: Didn't think of this earlier, but it'd be nice to add an example to this repo's README.

Comment (mkangia) [reply CHANGES.md]: I will add.

**Pull request merged**